src/xdocs/config_imports.xml

```diff
@@ -864,6 +864,65 @@ public class InputIllegalImport { }
     &quot;-//Puppy Crawl//DTD Import Control 1.2//EN&quot;
     &quot;http://checkstyle.sourceforge.net/dtds/import_control_1_2.dtd&quot;&gt;
         </pre>
+
+        <h4 id="blacklist-example">Example of blacklist mode</h4>
+        <p>
+          Short description of the behaviour:
+          <ul>
+            <li>Check starts checking from the longest matching sub-package to the current subpackage described inside import control file.</li>
```

Comment:
After come pause with this Check..... I do not not understand this sentence.

To my mind should be:
"Check starts checking from the longest matching subpackage described inside import control file to package defined in class file (later 'current subpackage')."